examples/basic/main.tf

```diff
@@ -15,6 +15,7 @@ module "expel_aws_cloudtrail_integration" {
   source = "../../"
 
   expel_customer_organization_guid = var.expel_customer_organization_guid
+  customer_aws_account_id          = var.customer_aws_account_id
```

Comment:
we use `expel_customer...` above -- for consistency sake, we should change this line to `expel_customer...` as well.